circle.yml

```diff
@@ -21,8 +21,10 @@ deployment:
     production:
         tag: /v[0-9]+(\.[0-9]+)*/
         commands:
-            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/$CIRCLE_TAG/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/latest/docs/ --delete
     develop:
         branch: master
         commands:
-            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/dev/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/dev/$CIRCLE_TAG/docs/ --delete
```

Comment:
So @seanrankine and I discussed `$CIRCLE_TAG/docs` vs `docs/$CIRCLE_TAG` and concluded that the former is easier to implement. 

@amongil Do you have time to make these changes? Dw if not - I can open up a PR myself but I'm currently blocked by this piece. Thanks!